awscli/examples/apigatewayv2/put-routing-rule.rst

```diff
@@ -0,0 +1,52 @@
+**To update a routing rule**
+
+The following ``put-routing-rule`` example updates the priority of a routing rule. ::
+
+    aws apigatewayv2 put-routing-rule \
+        --domain-name 'regional.example.com' \
+        --priority 150 \
+        --conditions '[
```

Comment:
Same here, lines 8-24 need to be condensed to a single lined or every line needs to be escaped.